packages/typespec-ts/src/transform/transformResponses.ts

```diff
@@ -141,6 +142,9 @@ function transformHeaders(
         SchemaContext.Output
       ]) as Schema;
       const type = getTypeName(typeSchema);
+      if (typeSchema.type === "object" && typeSchema.enum) {
```

Comment:
Can we use the getImportedModelName instead?

https://github.com/Azure/autorest.typescript/pull/2155/files#diff-71234ca40517891bd095b50338b2302c0fc737b72d7514989d13521ae67475d4R144-R147

Reply:
updated.